src/fetchers/top-languages-fetcher.js

```diff
@@ -42,10 +42,15 @@ const fetcher = (variables, token) => {
 /**
  * @param {string} username
  * @param {string[]} exclude_repo
+ * @param {string[]} include_repo
  * @returns {Promise<import("./types").TopLangData>}
  */
-async function fetchTopLanguages(username, exclude_repo = []) {
+async function fetchTopLanguages(username, exclude_repo = [], include_repo=[]) {
   if (!username) throw new MissingParamError(["username"]);
+  if (exclude_repo.length != 0 && include_repo.length != 0)
+  throw Error(
```

Comment:
@anuraghazra We can also use [CustomError](https://github.com/anuraghazra/github-readme-stats/blob/de05a574b145ffa6d2813c4fa53bc4fca67cb732/src/common/utils.js#L268-L281) here.